template.rb

```diff
@@ -8,31 +8,31 @@
 # This template assumes you have followed the setup guide at:
 # https://github.com/wildland/guides/#setting-up-your-development-enviroment
 #
-ruby_version = '2.2.3'
-node_version = 'v5.4.0'
+ruby_version = '2.3.1'
+node_version = 'v4.5.0'
```

Comment:
https://github.com/nodejs/LTS#lts-schedule and https://devcenter.heroku.com/articles/nodejs-support#supported-runtimes.  
TL;DR 5 isn't included in LTS. 
We could bump forward to 6.x.x if you think that makes more sense.